geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/index/FilterHelperTest.scala

```diff
@@ -353,4 +356,15 @@ class FilterHelperTest extends Specification with Mockito with Logging {
       result.asInstanceOf[KnownCost].cost mustEqual 5
     }
   }
+
+  "tryMergeGeoms" should {
+    "intersect BBOX geoms" >> {
```

Comment:
When the viewport spans the IDL and the request has a CQL filter, what happens? Does geoserver produce two Or queries?  i.e. (One side of IDL AND CQL) OR (Other side of IDL AND CQL)?